x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/components/api_logs_table.scss

```diff
@@ -0,0 +1,3 @@
+.apiLogDetailButton {
+  height: $euiSizeL !important;
```

Comment:
With:
<img width="1082" alt="" src="https://user-images.githubusercontent.com/549407/113223371-df2f2000-923d-11eb-8942-ef0f1b4dd1fa.png">

Without:
<img width="1088" alt="" src="https://user-images.githubusercontent.com/549407/113223378-e22a1080-923d-11eb-9408-22995ca2f19d.png">

I was trying to maintain the more compact rows from our standalone UI (as well as matching our other tables), but if we think that's overkill I can get rid of it.

FWIW without the `! important` I need 3 levels of nesting to override EUI's styles, so I took the lazy way out 😬

Reply:
If it were me, I'd say just let EUI do it's thing. However, if there's something unique about this table that is forcing the rows to be taller than usual, then I'd say this is a worthwhile change.

It's up to you though. Is it worthwhile to drop a comment in the CSS about why you have added that or nah?

To be clear, I'm good either way, I'll approve it regardless.

Reply:
The first screenshot more closely matches the row heights in all their examples. The main difference is that their buttons are all icon buttons and not empty buttons 🤷 It does seem odd that EUI wouldn't have any affordance OOTB for EuiEmptyButton matching the same line height as an EuiLink.

A comment makes a ton of sense! I'll add one here in a bit.

Reply:
https://github.com/elastic/kibana/pull/96008/commits/cf897c4aaef858b71d6d8e13022d527c17ace85f